src/os/ec_bsd.c

```diff
@@ -136,7 +136,7 @@ u_int16 get_iface_mtu(const char *iface)
 
    /* open the socket to work on */
    sock = socket(PF_INET, SOCK_DGRAM, 0);
-   if (sock = -1)
+   if (sock == -1)
```

Comment:
@koeppea basically you are saying that our BSD port never worked? :)
or at least the get mtu never did ;)